CRM/Activity/Form/Activity.php

```diff
@@ -528,7 +528,7 @@ public function preProcess() {
     if ($this->_action & CRM_Core_Action::UPDATE) {
       // We filter out alternatives, in case this is a stored e-mail, before sending to front-end
       if (isset($this->_values['details'])) {
-        $this->_values['details'] = CRM_Utils_String::stripAlternatives($this->_values['details']) ?: '';
+        $this->_values['details'] = CRM_Utils_String::stripAlternatives(CRM_Utils_Array::value('details', $this->_values)) ?: '';
```

Comment:
So this is an e-notice fix but you should probably simplify to 
CRM_Utils_Array::value('details', $this->_values, '');

rather than use the ternery AND the Array fn

for readability

Reply:
@seamuslee001 got in and fixed the php notice with isset above so we don't need this change anymore - I've removed it.